docs/manuals/core/entities/scheduling-and-job-execution.md

```diff
@@ -1,26 +1,50 @@
-# Submit a scenario, pipeline or task.
+# Submit a scenario, pipeline, or task
+
+In a Taipy application, running the Core service is required to execute jobs.
+
+To see how you can run different Taipy services, please refer to the [Running Taipy services](../../running_services/index.md) page.
+
+!!! Important "Block configuration after running Taipy Core services"
+    After running the Core services, all configuration will be blocked from update to prepare for job execution.
 
 To execute a scenario, you need to call the `taipy.submit()^` method:
 
 ```python linenums="1"
 import taipy as tp
 import my_config
 
+tp.Core().run()
+
 scenario = tp.create_scenario(my_config.monthly_scenario_cfg)
 
 tp.submit(scenario)
 ```
 
-In line 4, we create a new scenario from a scenario configuration and submit it for execution (line 6). The `submit`
+In line 6, we create a new scenario from a scenario configuration and submit it for execution (line 8). The `submit`
 method triggers the submission of all the scenario's pipelines. Then each task of each pipeline will be submitted.
 
+The Core service can also be started after `submit` method. Note that jobs can only be executed after the Taipy Core service is started.
```

Comment:
- "submit" is not a method. It's a function.
- Methods or function should be referenced with their () parenthesis (and potentially parameter names or values)
- Make sure it is cross-referenced (`(taipy.)submit()^`).